src/cmds/plots.py

```diff
@@ -34,9 +34,6 @@ def make_parser(parser):
     parser.add_argument(
         "-n", "--num", help="Number of plots or challenges", type=int, default=None
     )
-    parser.add_argument(
-        "-i", "--index", help="First plot index", type=int, default=None
```

Comment:
This has already been removed in 1.8 as per bram's request. We can replot deterministically by passing in optional "test_private_keys" to create_plots. This is called from the tests (block_tools.py), but it is impossible from the command line, to prevent people from reusing plot keys.